composer.json

```
@@ -5,7 +5,8 @@
     "license": "GPL-3.0-or-later",
     "require": {
         "neos/redirecthandler": "~3.0 || ~4.0 || ~5.0 || dev-master",
-        "neos/neos": "~4.3 || ~5.0 || ~7.0 || ~8.0 || dev-master"
+        "neos/neos": "~9.0 || dev-master",
+        "neos/flow": "~9.0 || dev-master"
```

We no longer have a `master` branch (nor a `main` branch.)
```suggestion
        "neos/neos": "^9.0",
        "neos/flow": "^9.0"
```